src/ccb_neb/service_group_member.cc

```diff
@@ -31,8 +31,22 @@ using namespace com::centreon::broker::neb;
  *  Default constructor.
  */
 service_group_member::service_group_member()
-    : group_member(service_group_member::static_type()), service_id(0) {}
+    : group_member(service_group_member::static_type(), "", 0, 0, 0, true), service_id(0) {}
 
+service_group_member::service_group_member(std::string const& group_name,
+                                           uint32_t group_id,
+                                           uint32_t poller_id,
+                                           uint32_t host_id,
+                                           bool enabled,
+                                           uint32_t service_id)
```

Comment:
It's a good practice to keep host_id and service_id together, so could you exchange enabled and service_id in the constructor please?